lib/mayaHydra/hydraExtensions/mayaHydraSceneProducer.cpp

```diff
@@ -54,11 +54,9 @@ MayaHydraSceneProducer::MayaHydraSceneProducer(
         _sceneIndex = MayaHydraSceneIndex::New(initData, lightEnabled);
         TF_VERIFY(_sceneIndex, "Maya Hydra scene index not found, check mayaHydra plugin installation.");
         
-        //Create an HydraViewportInformation instance
-        FVP_NS::InformationInterface::ViewportInformation* hydraViewportInformation = 
-            new FVP_NS::InformationInterface::ViewportInformation(_sceneIndex, initData.cameraName, initData.viewportWidth, initData.viewportHeight, initData.rendererName);
-        //This method takes the ownership of the hydraViewportInformation passed to it and will delete it
-        FVP_NS::ViewportInformationAndSceneIndicesPerViewportDataManager::Get().AddViewportInformation(hydraViewportInformation, _renderIndexProxy);
+        //Create an HydraViewportInformation 
+        const Fvp::InformationInterface::ViewportInformation hydraViewportInformation(_sceneIndex, initData.cameraName, initData.viewportWidth, initData.viewportHeight, initData.rendererName);
```

Comment:
Unfortunate that we need to expand out MayaHydraDelegate::InitData to convert between two similar data structures.  Might want to eventually simply use Fvp::InformationInterface::ViewportInformation instead of MayaHydraDelegate::InitData, to reduce code complexity.